hardhat.config.ts

```diff
@@ -1,99 +1,21 @@
-import * as dotenv from "dotenv";
-
-import { HardhatUserConfig } from "hardhat/config";
-import { getNodeUrl, getMnemonic } from "@uma/common";
-
-// Custom tasks to add to HRE.
-// FIXME: Temporarily commenting out tasks to minimize amount of files imported and executed at compile time
-// to construct the Hardhat Runtime Environment. Generally we'd prefer to keep the HRE construction
-// lightweight and put runnable scripts in the `scripts` directory rather than add as an HRE task.
-// TODO: Figure out which imported module in `./tasks` is causing HRE construction to fail.
-// require("./tasks");
-
 import "@nomiclabs/hardhat-etherscan";
 import "@nomiclabs/hardhat-waffle";
+import "@openzeppelin/hardhat-upgrades";
 import "@typechain/hardhat";
+import "hardhat-deploy";
 import "hardhat-gas-reporter";
+import { HardhatUserConfig } from "hardhat/config";
 import "solidity-coverage";
-import "hardhat-deploy";
-import "@openzeppelin/hardhat-upgrades";
-
-dotenv.config();
 
 const solcVersion = "0.8.18";
-const mnemonic = getMnemonic();
 
 const config: HardhatUserConfig = {
```

Comment:
Yeah I don't think we need these extra networks for the SDK since we only want to run unit tests. The relayer's `hardhat.config` is the one where we want all the networks.

Reply:
Removed